src/main/java/lach_01298/qmd/accelerator/Accelerator.java

```diff
@@ -79,7 +79,7 @@
 	
 	public List<ParticleStorageAccelerator> beams = Lists.newArrayList(new ParticleStorageAccelerator(),new ParticleStorageAccelerator());
 	
-	public boolean refreshFlag = true, isAcceleratorOn = false, cold = false;
+	public boolean refreshFlag = true, isAcceleratorOn = false, cold = false, manualDisableFlag = false, manualEnableFlag = false;
 	

```

Comment:
what is the point of having 2 flags?

Reply:
Well. Yeah it could be done with one overrideFlag
am using this flags here

if ((isRedstonePowered() || getAccelerator().manualEnableFlag) && !getAccelerator().manualDisableFlag)